doc/classes/SpringBoneSimulator3D.xml

```diff
@@ -378,8 +378,8 @@
 			<param index="0" name="index" type="int" />
 			<param index="1" name="enabled" type="bool" />
 			<description>
-				If sets [param enabled] to [code]true[/code], the all child [SpringBoneCollision3D]s are collided and [method set_exclude_collision_path] is enabled as an exclusion list at [param index] in the settings.
-				If sets [param enabled] to [code]false[/code], you need to manually register all valid collisions with [method set_collision_path].
+				If [param enabled] is [code]true[/code], all the child [SpringBoneCollision3D]s are collided and [method set_exclude_collision_path] is enabled as an exclusion list at [param index] in the settings.
```

Comment:
```suggestion
				If [param enabled] is [code]true[/code], all child [SpringBoneCollision3D]s are colliding and [method set_exclude_collision_path] is enabled as an exclusion list at [param index] in the settings.
```
Or
```suggestion
				If [param enabled] is [code]true[/code], all child [SpringBoneCollision3D]s have collided and [method set_exclude_collision_path] is enabled as an exclusion list at [param index] in the settings.
```

Reply:
I have no idea what "are collided" even means here so I kept it safe for this PR

Reply:
I'd assume it means all of them have finished colliding or similar, but it's ungrammatical this way